### ISSUE TITLE
Vendor Login #

### ISSUE THREAD

**Issue by JasonSchneider19** [Sanjel-Energy-Services/LabDatabase_WorkItems#68]
Add a field for "Vendor Login #" in the Sample Login as follows:

![Image](https://user-images.githubusercontent.com/95711838/191080280-2ff86425-eabf-494d-b60c-f0eb88c4a6ad.png)

**Comment by tobeytong** [comment]
In this form, We can see that the Login # field has been displayed in the first column of the first row，Why do us still need add a "Login #" in the red line box?Are they from different tables? If yes, please let us know

**Comment by awangatsanjel** [comment]
@JasonSchneider19 Is the new "login #" acutally the sample number on the bag from Bulk Plant?

**Comment by JasonSchneider19** [comment]
@tobeytong @awangatsanjel 
This should be "Lot #", not "Login #".
The Lot # will be populated by the laboratory staff


![Image](https://user-images.githubusercontent.com/95711838/204052941-fcf64c49-f9e8-4cd6-938d-9d249a0463cf.png)

**Comment by awangatsanjel** [comment]
@JasonSchneider19 @cgaine 

There is no Lot field in current Lab Sample UI. There is a field in Lab Sample entity named "StorageLocation". Was it for this lot purpose?

**Comment by JasonSchneider19** [comment]
@awangatsanjel 
@cgaine 
no this is new. This would be a serial # that comes from the supplier, or chemical bag, or an manufacturer identifier

**Comment by cgaine** [comment]
Should identify this as Vendor login number , especially in the database, to avoid confusion

**Comment by JasonSchneider19** [comment]
@awangatsanjel 
Lab has requested this label be changed to "Batch #"

**Comment by wistaria623** [comment]
@awangatsanjel 
Does the naming in the entity definition need to be modified synchronously?

**Comment by awangatsanjel** [comment]
No, Label only for now. Entity naming will be determined after more information is provided.
